terraform/modules/self-service/modules/cognito/cognito.tf

```diff
@@ -2,9 +2,94 @@ locals {
   service = "self-service"
 }
 
+resource "aws_sns_topic" "cognito-sns-topic" {
+  name = "cognito-sns-topic"
+}
+
+resource "aws_sns_topic_policy" "cognito-sns-policy" {
+  arn = "${aws_sns_topic.cognito-sns-topic.arn}"
+
+  policy = "${data.aws_iam_policy_document.sns-topic-policy.json}"
+}
+
+data "aws_iam_policy_document" "sns-topic-policy" {
+  policy_id = "__default_policy_ID"
```

Comment:
is this required?

Reply:
The policy document is required... yes

Reply:
policy document yes, but the `policy_id` was optional I thought